### DISCUSSION TITLE
Listen on Specific IP Address

### DISCUSSION THREAD

**Discussion by tadscottsmith** [chuot/rdio-scanner#386] [Q&A]
Has anyone setup rdio-scanner to only listen on one specific interface/IP address? I have a web host that I'd like to point multiple domains to and have separate applications respond on port 443. I believe this could be accomplished by using Docker, but I'm curious if anyone has done it with the native application?

**Comment by atlex** [comment]
./rdio-scanner  -listen ":443"

./rdio-scanner -h
